src/encoded/static/components/static-pages/components/StatisticsPageViewBody.js

```diff
@@ -0,0 +1,1049 @@
+            let groupingKey = "ga:dimension5"; // Experiment Type
+            const countKey = 'ga:metric2'; // Download Count
+
+            if (countBy === 'filetype'){
+                useReport = 'file_downloads_by_filetype';
+                groupingKey = 'ga:productVariant'; // File Type
+            } else if (countBy === 'geo_country'){
+                useReport = 'file_downloads_by_country';
+                groupingKey = 'ga:country';
+            }
+
+            console.log("AGGR", resp, props, countBy, groupingKey, useReport);
+
+            //if (props.file_downloads_by_experiment_type_group_by === 'term') groupingKey = 'ga:dimension4';
+            //if (props.file_downloads_by_experiment_type_group_by === 'field+term') groupingKey = 'ga:eventLabel';
+
+            return commonParsingFxn.analytics_to_buckets(resp, useReport, groupingKey, countKey);
+
+            // if (!resp || !resp.aggregations || !props.countBy || !props.countBy.file_downloads) return null;
+            // const dateAggBucket = props.currentGroupBy && (props.currentGroupBy + '_interval_date_created');
+            // const dateIntervalBuckets = resp && resp.aggregations && resp.aggregations[dateAggBucket] && resp.aggregations[dateAggBucket].buckets;
+
+            // if (!Array.isArray(dateIntervalBuckets) || dateIntervalBuckets.length < 2) return null;
+
+            // return commonParsingFxn.countsToCountTotals(
+            //     commonParsingFxn.bucketDocCounts(dateIntervalBuckets, props.countBy.file_downloads)
+            // );
+        }
+    },
+};
+
+// I forgot what purpose of all this was, kept because no time to refactor all now.
+export const submissionsAggsToChartData = _.pick(aggregationsToChartData,
+    'expsets_released', 'expsets_released_internal',
+    'expsets_released_vs_internal', 'files_released',
+    'file_volume_released'
+);
+
+export const usageAggsToChartData = _.pick(aggregationsToChartData,
+    'sessions_by_country', 'fields_faceted', 'experiment_set_views', 'file_downloads'
+);
+
+
+export class UsageStatsViewController extends React.PureComponent {
+
+    static getSearchReqMomentsForTimePeriod(currentGroupBy = "daily"){
+        const untilDate = moment.utc();
+        let fromDate;
+        if (currentGroupBy === 'monthly'){ // 1 yr (12 mths)
+            untilDate.startOf('month').subtract(1, 'minute'); // Last minute of previous month
+            fromDate = untilDate.clone();
+            fromDate.subtract(12, 'month'); // Go back 12 months
+        } else if (currentGroupBy === 'daily'){ // 30 days
+            untilDate.subtract(1, 'day');
+            fromDate = untilDate.clone();
+            fromDate.subtract(30, 'day'); // Go back 30 days
+        }
+        return { fromDate, untilDate };
+    }
+
+    static defaultProps = {
+        'searchURIs' : {
+            'TrackingItem' : function(props) {
+                const { currentGroupBy, href } = props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['href' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5603033)